### PR TITLE
Shubham dalal work

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

**Review by Ashwin2812 — APPROVED** [review]

File: Shubham_Dalal/HTML/ASSIGNMENT1/shubhamforth.html [hunk 1]
```diff
@@ -4,10 +4,9 @@
...
     <body>
-        <h1>Introduction</h1>
-       
+        <h1>Introduction</h1>    
         <p>My Name is Shubham Dalal. I'm from Burhanpur MP. I have done my Engineering in Computer Science from <br>
-         Thakur Shivkumar Singh Memorial Engineering College.</p>
-    
+         Thakur Shivkumar Singh Memorial Engineering College.
+        </p>  
         </body>
```
Comment (Pawan1005) [inline Shubham_Dalal/HTML/ASSIGNMENT1/shubhamforth.html]: check indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
check tab setting we need 4 space only your files contain 7-8 spaces

**Review by nileshp1122 — COMMENTED** [review]
check indentation

File: Shubham_Dalal/HTML/ASSIGNMENT1/shubhamforth.html [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+		<title>second assignment</title>
+	</head>
+	<body>
+		<h1>Introduction</h1>
+		<p>
+			My Name is Shubham Dalal. I'm from Burhanpur MP. I have done my Engineering in Computer Science from <br />
+			Thakur Shivkumar Singh Memorial Engineering College.
+		</p>
+	</body>
+</html>
```
Comment (nileshp1122) [inline Shubham_Dalal/HTML/ASSIGNMENT1/shubhamforth.html:13]: check indentation

**Review by nileshp1122 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

File: Shubham_Dalal/HTML/ASSIGNMENT2/shubham_style.html [hunk 3]
```diff
@@ -1,36 +1,34 @@
...
-    <center style="color: slategrey; font-size: 25px;">Thankyou!</center>
-</body>
-
-</html>
\ No newline at end of file
+    <head>
+        <title>My first style page</title>
+    </head>
+    <body style="background-color: darksalmon">
+        <center>
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT2/shubham_style.html]: dont use center tag instead use text-align: center property

File: Shubham_Dalal/HTML/ASSIGNMENT2/shubham_style.html [hunk 4]
```diff
@@ -1,36 +1,34 @@
...
+            individual pages on the <b style="color: darkcyan">World Wide Web</b>
+        </p>
+        <p>They called this search engine <i style="background-color: lightpink">Backrub</i>.</p>
+        <h1>Initial public offering</h1>
+        <p>
+            Google's initial public offering (IPO) took place five years later, on 19th August 2004. <br />
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins> <br />
+            years, until the year 2024.
+        </p>
+        <center style="color: slategrey; font-size: 25px">Thankyou!</center>
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT2/shubham_style.html]: dont use center tag insted use text-align: center property

File: Shubham_Dalal/HTML/ASSIGNMENT3/shubham_contactus.html [hunk 5]
```diff
@@ -1,46 +1,40 @@
...
+            }
+            a:visited {
+                color: darkgray;
+            }
+            a:hover {
+                color: rgb(65, 223, 215);
+            }
+            </style>
+    </head>
+    <body style="text-align: center; background: rgb(71, 99, 22)">
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT3/shubham_contactus.html:22]: check indentation regarding spaces

File: Shubham_Dalal/HTML/ASSIGNMENT4/Shubham_table1.html [hunk 6]
```diff
@@ -1,57 +1,51 @@
...
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>First Table</title>
+        <style>
+            table {
+                width: 100%;
+                background-color: rgb(173, 182, 190);
+            }
+            table,
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT4/Shubham_table1.html]: include table, th, tr, td {} like this on same line

File: Shubham_Dalal/HTML/ASSIGNMENT4/shubham_table2.html [hunk 7]
```diff
@@ -1,104 +1,93 @@
...
+    <head>
+        <meta charset="UTF-8"/>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <title>Second Table</title>
+        <style>
+            h2 {
+                color: steelblue;
+            }
+            table,
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT4/shubham_table2.html]: include all tag names in a single line

File: Shubham_Dalal/HTML/ASSIGNMENT4/shubham_table2.html [hunk 8]
```diff
@@ -1,104 +1,93 @@
...
+            th,
+            td {
+                border-collapse: collapse;
+            }
+            table {
+                border: 8px solid slategray;
+            }
+            th,
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT4/shubham_table2.html]: include th and td in a single line and add space after comma always

File: Shubham_Dalal/HTML/ASSIGNMENT4/shubham_table3.html [hunk 9]
```diff
@@ -1,91 +1,83 @@
...
-
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8"/>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <title>Third Table</title>
+        <style>
+            table,
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT4/shubham_table3.html]: include all tags in a single line

File: Shubham_Dalal/HTML/ASSIGNMENT4/shubham_table3.html [hunk 10]
```diff
@@ -1,91 +1,83 @@
...
+            th,
+            td {
+                border: 1px solid cornflo    erblue;
+                text-align: center;
+                background-color: rgb(233, 223, 207);
+                margin: 100px 400px;
+            }
+            th,
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT4/shubham_table3.html]: include th, td {} in a single line

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

File: Shubham_Dalal/HTML/ASSIGNMENT4/Shubham_table1.html [hunk 11]
```diff
@@ -1,57 +1,51 @@
...
+            tr,
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT4/Shubham_table1.html]: do the change in this file also

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do change proper

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**